### ISSUE TITLE
Pipelines need to support target namespace as a parameter

### ISSUE THREAD

**Issue by nastacio** [kabanero-io/kabanero-pipelines#290]
While attempting to build a new appsody application using a kabanero pipeline, I notice that all applications end up getting deployed to the same cluster namespace where the pipeline is running, without an option to indicate the desired target namespace for the application.

As a workaround I see some app developers hardcoding the target namespace of a specific cluster into the app-deploy.yaml file of the appsody application and checking it into the Git repo for the app (!) . That means applications must be hardcoded to one specific target deployment, which means applications cannot be reused anywhere else unless the administrators for all target clusters agree on the namespaces for all applications that must be connected to one another and reconfigure all of their specific operators (E.g. Istio) to work on all those namespaces.

In short, requiring applications to hardcode namespaces make them virtually unusable, so we need the pipelines to accept the target namespace as a parameter.

**Comment by kvijai82** [comment]
Our plan is provide a set of deploy pipelines that will work with gitops in the near future.  I think that will address the concerns that you have raised.

**Comment by kvijai82** [comment]
Plan is to address this via gitops by work in this epic: https://github.com/kabanero-io/kabanero-pipelines/issues/255
